src/Nbody/Sinks.cpp

```diff
@@ -463,6 +461,29 @@ void Sinks<ndim>::AccreteMassToSinks
     //---------------------------------------------------------------------------------------------
 
 
+    // Determine the closest sinks to all particles that are inside sinks
+#pragma omp for schedule(dynamic,1)
```

Comment:
Hmmm, maybe remove, or maybe have conditional parallelisation only if Nsinks is greater than some threshold?  Either way, I don't mind so much.